### ISSUE TITLE
Problem of Segmentation Fault

### ISSUE THREAD

**Issue by zsh2000** [marcomusy/vedo#412]
Hi,

I want to use your library "vedo" to visualize human motions, but I find that when running the line "vedo.show(...)", a "segmentation fault (core dumped)" will always occur. 

When I run the following example code on https://vedo.embl.es/, this problem still exists:

```
from vedo import *
s = Sphere()
c = Cube()
p = Paraboloid()
show(s, c, at=[0, 1], shape=(3,1))
show(p, at=2, interactive=True)
```

Plus, I have tried running the code in both the environment of Python 3.8 and Python 3.7, but both of them can't escape from this problem.

Sincerely,
Shuhong

**Comment by marcomusy** [comment]
Hi , the above works in my mac. I guess something is wrong in your vtk installation.
Can you try running this script:
https://gitlab.kitware.com/vtk/vtk/blob/573ad7a3d176e569a937caa4248118fd01e5e8e5/Examples/Tutorial/Step1/Python/Cone.py

**Comment by zsh2000** [comment]
Hi,

Thanks for you quick reply!

I tried running the script you gave and gives the following error message (running on Ubuntu 18.04.4):

<img width="760" alt="微信截图_20210610194040" src="https://user-images.githubusercontent.com/52485301/121519173-f7809f80-ca23-11eb-8584-2d35a57cf10c.png">

The "segmentation fault" still occurs.

I tried to install a valid OpenGL version by "pip install", but the latest version I could get is 3.1.5.
Is the "segmentation fault" caused by the "out-of-date OpenGL"?

Sincerely,
Shuhong
